documentation/API_documentation/QoD_API.md

```diff
@@ -26,14 +26,14 @@ Security access keys such as OAuth 2.0 client credentials used by client applica
 **QoS profiles and QoS profile labels**
 Latency or throughput requirements of the application mapped to relevant QoS profile class.
 
-**Identifier for the the user equipment (UE)**
-At least one identifier for the user equipment out of four options: IPv4 address, IPv6 address, MSISDN, or external ID [[5]](#5) assigned by the mobile network operator for the user equipment
+**Identifier for the the device**
```

Comment:
typo: the the

Reply:
Done in https://github.com/camaraproject/QualityOnDemand/pull/129/commits/fe2cef99b05112008c4d5f59316cbfb4e1b1dd85